README.md

```diff
@@ -45,13 +45,16 @@ doing at your convenience.
 
 If you are using CMake and want to use its configuration abilities to save
 you the trouble, you can easily integrate Backward:
+
 ```
-add_subdirectory(/path/to/backward-cpp)
+find_package(Backward)
```

Comment:
> Damn this cmake configuration is getting out of hand. We; programmers; have really fucked up our build systems.

Yes, I agree :(

It should support, yes. Let me test this to be 100% sure and I will get back to you.

As I'm packing the library for conda-forge I noticed that I may revert the `find_package()` change at CMakeLists.txt, since it doesn't make much sense the way I changed it.